### PR TITLE
Moderation page link

### PR DESCRIPTION
**Pull request by Raemon**
Adds a link to the moderation page at the bottom of post pages.

My guess is EAForum doesn't currently want this (they had previously hidden the moderation page from non-admins since it otherwise wasn't very discoverable and some people found it jarring to find out their moderation decisions were public). I can either forum-gate this for EA Forum, or remove the admin-gate for the /moderation page. 

Also fixes a bug in the /moderation page where people who'd only banned users from their personal blogposts weren't showing up. 

image

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1203897826008742) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by Raemon** [comment]
Assigning to @jpaddison3 since I assume whether to forum-gate is more his-call.

**Review by oetherington — APPROVED** [review]
Approving because the code looks good to me, but we should still wait for JP to answer on forum-gating before merging.

**Comment by jpaddison3** [comment]
Um, talk me through this. 

(FWIW, https://forum.effectivealtruism.org/moderation is still public on the EA Forum, but with the section of individuals' moderation decisions removed, for the reason you mention.)

I can guess that y'all want to increase the amount of links to moderation, that seems good. But this seems like an odd place.

I could imagine this being a good feature for LW if it checked "have there been any moderation actions on this post" before displaying. Though for the EA Forum, it'd be more dicey because of the lack of personal moderation content. Honestly though if it checked that, that would be a good way to break the awkward semi-public nature of personal moderation, and I might consider reenabling that section of the page.

**Comment by Raemon** [comment]
So my reasoning is:

– at the bottom of comment sections seems like a reasonable place to make people aware "moderation actions happen sometimes, here's the list of all of them." Comment sections are the place you might want to make comments that might get moderated.

– I prefer showing it in all comment threads rather than "just places that have received a moderation action", because I don't want people making moderation actions to have to draw extra attention to themselves. (This reasoning is a bit cached. I think I still endorse it – by default I still think people are too afraid to moderate, and barely do so, and having the site draw extra attention to it nudges towards a world where any moderation will cause you enough stress to outweigh what you were hoping to gain via moderation. But, I think Ruby may not even necessarily agree with me here)

**Comment by jpaddison3** [comment]
I see where your coming from now. I might be interested to see it play out, but I predict I would still disagree afterwords.

Interesting point re second bullet point. I don't have a particularly strong view there for LW now. On the EA Forum, I would still endorse conditionally showing it. We want to be light-touch with moderation anyway.

**Comment by Raemon** [comment]
Can I ship it as-is, and you build a conditional thing if you want?

**Review by darkruby501 — APPROVED** [review]

**Comment by jpaddison3** [comment]
Forum-gate it, and we'll see how it plays out!

File: packages/lesswrong/lib/collections/users/views.ts
Comment (jpaddison3) [inline packages/lesswrong/lib/collections/users/views.ts:113]: Did you know that this would have been a bug until very recently?

Comment (jpaddison3) [reply packages/lesswrong/lib/collections/users/views.ts:113]: _Very_ recently: https://github.com/ForumMagnum/ForumMagnum/pull/6618/files#diff-dff188afe3a43c2ed4ddd6a3daa618395937c99ab910204e89a70e15c0473a3b

Comment (jpaddison3) [reply packages/lesswrong/lib/collections/users/views.ts:113]: Or maybe not, depends on if the default view of users has an `$or`, which, do I remember that there isn't one?

Comment (Raemon) [reply packages/lesswrong/lib/collections/users/views.ts:113]: I... did not know that. (I did start by filtering for "exists: true" which explicitly didn't work for bannedPersonalUserIds for some reason)

File: packages/lesswrong/components/comments/CommentsListSection.tsx
Comment (jpaddison3) [inline packages/lesswrong/components/comments/CommentsListSection.tsx:20]:
```suggestion
```

File: packages/lesswrong/components/comments/CommentsListSection.tsx
Comment (jpaddison3) [inline packages/lesswrong/components/comments/CommentsListSection.tsx:16]:
```suggestion
import { isEAForum } from '../../lib/instanceSettings';
```

**Pull request merged**
